include/time.h

```diff
@@ -25,15 +25,19 @@
 #ifndef KTF_TIME_H
 #define KTF_TIME_H
 
+#define MS(s) (s * 1000)
+#define uS(s) (MS(s) * 1000)
+#define NS(s) (uS(s) * 1000)
```

Comment:
If we really want these macros -- they're very generic after all -- please add `ULL` to the constants to make sure they're treated as 64 bit integers. Otherwise even small `NS(...)` values will overflow the default `int` type.

```C
#define MS(s) (s * 1000ULL)
#define uS(s) (MS(s) * 1000ULL)
#define NS(s) (uS(s) * 1000ULL)
```

Reply:
Yes. Dropping the macros.